python/main/config/config.py

```diff
@@ -19,8 +19,65 @@ class HatcheryConfig:
     facility: str
     wria: str
     river_gauge: str
+    lon: float
     lat: float
-    long: float
+    reduced_coordinate_key: int
+
+
+# lon, lat
+reduced_coordinates_dict = {
+    0: (-121.6981, 48.6491),
+    1: (-123.3287, 46.2259),
+    2: (-123.4003, 47.1461),
+    3: (-124.4389, 47.9349),
+    4: (-121.9625, 47.3761),
+    5: (-120.6519, 47.789),
+    6: (-117.2966, 46.0376),
+    7: (-122.6293, 46.5113),
+    8: (-123.1406, 48.028),
+    9: (-120.2892, 47.5336),
+    10: (-123.5493, 48.1152),
+    11: (-123.5948, 46.558),
+    12: (-122.5741, 47.1915),
+    13: (-123.1818, 47.3013),
+    14: (-123.1399, 47.407),
+    15: (-123.9892, 47.2332),
+    16: (-123.1433, 48.1197),
+    17: (-121.9786, 47.2799),
+    18: (-122.0386, 47.5295),
+    19: (-122.142, 48.8973),
+    20: (-122.6165, 45.937),
+    21: (-118.2287, 46.5969),
+    22: (-121.4178, 48.5223),
+    23: (-123.203, 47.3066),
+    24: (-122.7026, 47.3726),
+    25: (-123.7531, 46.3722),
+    26: (-123.8411, 46.503),
+    27: (-122.572, 46.3746),
+    28: (-119.8967, 46.6486),
+    29: (-121.6241, 47.839),
+    30: (-119.2612, 46.5146),
+    31: (-122.3315, 48.5649),
+    32: (-122.2179, 45.6208),
+    33: (-122.7255, 46.79),
+    34: (-122.1688, 47.3093),
+    35: (-122.4053, 45.9887),
+    36: (-121.8397, 47.5536),
+    37: (-117.6628, 46.3201),
+    38: (-122.9043, 47.0144),
+    39: (-122.1775, 47.0828),
+    40: (-122.166, 45.6518),
+    41: (-123.371964, 46.226517),
+    42: (-122.73316, 46.016026),
+    43: (-123.742635, 46.980079),
+    44: (-122.555653, 45.956563),
+    45: (-122.838527, 46.044736),
+    46: (-124.306113, 48.054483),
+    47: (-121.550793, 47.803818),
+    48: (-121.717, 47.8674),
+    49: (-122.255633, 45.618172),
+    50: (-121.720542, 48.27576),
+}
```

Comment:
This looks kinda wild but it'll let us reduce the number of calls to the weather API if we end up merging multiple hatcheries to a single coordinate

Reply:
Ape strong